### ISSUE TITLE
Synology no longer connects 

### ISSUE THREAD

**Issue by mikebutash** [zerotier/ZeroTierNAS#23]
I've used the synology addon for a few months, and noticed it doesn't work any longer.  I tried leaving the network, re-joining the network, but nothing works.  I deleted the object in the network dashboard, it never shows back up.  Even removed the package and reinstalled, still shows able to join ok at the the linux cli, but never shows up in the dash.  Logs show nothing at all useful short of start/stop of client.

Anyone else see this?  Using currently DSM 6.1.2-15132 and latest 1.2.8 zerotier modules for it.

**Comment by mikebutash** [comment]
Even updated to the latest DSM 6.1.7-15284, still doesn't work anymore with a fresh join or re-install.  Sad, I've come to rely on this working, not sure the change in things.

**Comment by mikebutash** [comment]
Hi Joseph, I went and upgraded the package, but still seems mostly the same.  This is a Synology RS814RP+.

Specifically what I'm getting for the most part is it showing "ACCESS_DENIED PRIVATE' from the device.  I join, unjoin, remove the spk, add back the spk, repeat, nothing gets it past access_denied in the web ui.

**Comment by mikebutash** [comment]
Tonight I created a new account under this account domain, and I get the same thing with an unmolested default install when joining, access_denied.  Not sure why it doesn't like this client, maybe some trust issue?

**Comment by mikebutash** [comment]
So I created a new account, network, auto-assign prefix, and route for encompassing kit, and even the ui denies me adding a static address, something seems broken in the ui.  Any new host seems broken in existing, and new doesn't seem to work what so ever.  Message me if you'd like more specific account details.

**Comment by joseph-henry** [comment]
Hey @mikebutash Are you still seeing this issue? After re-reading your description of the problem I'm still uncertain what is happening exactly, it sounds like it might be multiple issues. The GUI for the Synology is definitely flaky at times.

Feel free to send me an email directly at joseph.henry@zerotier.com with your network ID so that I can take a look at its configuration. Hopefully we can resolve this.

**Comment by mikebutash** [comment]
Hi Joseph, I finally got this working again, but not using that existing network.  I ended up creating a new account, network, and migrating to that as it was an eventual goal anyways, and seems to be behaving there at least.  I never got any other indication of why it wasn't working, other than I was getting the permission denied for some reason.

I did remember a bit later that oddly before that, I found a new id connected I couldn't determine, so I had removed/blocked it, but wonder if maybe it somehow regenerated the id and that was it.  I couldn't find a way to see any removed/blocked id's to either verify or disregard entirely.

Part of the issue as well was moving it to the new network, it would connect, but not get an address.  I found this was consistent later adding other hosts, where zt didn't seem to want to work using a /21 subnet for my network auto-assign poo.  I bumped it down to a /22, and it suddenly began handing out addresses, including to my synology(s).  That was a bit weird, maybe something for another bug report, but otherwise working now.

Thanks!
